.github/workflows/check.yml

```diff
@@ -241,26 +167,23 @@ jobs:
         submodules: true
     - name: Create logs dir
       run: mkdir test-logs
-    - name: autoreconf
-      run: autoreconf -fi
     - name: configure
       run: |
-        mkdir _build
-        ( cd _build && ../configure )
+        meson _build -Db_sanitize=address,undefined
```

Comment:
I don't think the valgrind test should enable sanitizers.

Reply:
Good catch. Thanks!